### PR TITLE
Fix npm URL

### PR DESCRIPTION
**Pull request by sarthak-sehgal**
Fix issue https://github.com/biojs/biojs-frontend/issues/33

### CODE REVIEW

**Comment by sarthak-sehgal** [comment]
Point guide to README as in issue https://github.com/biojs/biojs-frontend/issues/34 done

File: src/components/NavBar.vue [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <div class="navBar" :class="{ noShadow: !isShadow }">
 	<div id="nav" ref="nav">
 		<router-link class="navLink" to="/components"><span>Components</span></router-link>
-		<router-link class="navLink" to="/guide"><span>Guide</span></router-link>
+		<a class="navLink" href="https://github.com/biojs/biojs-frontend/blob/master/README.md" target="_blank"><span>Guide</span></a>
```
Comment (DennisSchwartz) [inline src/components/NavBar.vue]: Sorry I think there was a misunderstanding here. I didn't mean for the website to link to github. I thought we should move the content of the Guide section to the Readme. And replace the current content with a more high-level description about BioJS

Comment (DennisSchwartz) [reply src/components/NavBar.vue]: Also, can we have a separate PR for each issue? :) That way we can discuss them in their own place. If you remove this bit from here, we can merge it. And then we can talk about the Guide thing in it's own PR.

**Comment by sarthak-sehgal** [comment]
@DennisSchwartz I have reset the branch to the previous commit (fix npm URL). Please review.

**Comment by DennisSchwartz** [comment]
Thanks!

**Pull request merged**